nodejs/aws-infra/experimental/clusterAutoScaling.ts

```diff
@@ -0,0 +1,566 @@
+// Copyright 2016-2018, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+import * as mod from ".";
+
+import { Overwrite, sha1hash } from "./../utils";
+
+export class ClusterAutoScalingLaunchConfiguration extends pulumi.ComponentResource {
```

Comment:
Open question on the naming of these guys.  Should we just call it LaunchConfiguration since that's the relevant thing it will create.  Or should the name guide you wrt to expected intent.  i.e. this is to make a launch config intended to help cluster autoscale services within it?